cli/utilities/utils.py

```diff
@@ -320,3 +322,55 @@ def set_service_state(node, service_id, state):
     if err:
         return False
     return True
+
+
+def bring_node_offline(node, secondary_node, interface="eth0", timeout=120):
+    """
+    Brings a node offlibe by making network interface down for a defined time
+        Args:
+            node (ceph): Node at which the interface has to be bought down
+            secondary_node (ceph) : Another node in the cluster to ping and verify
+                                    the given node is down
+            interface (str): Node interface name ens3/eth0 etc.
+            timeout (int): Time duration (in secs) for which network has to
+                           be down
+        Returns (Thread): Thread object, None if failed
+        Example:
+            bring_node_offline(mon_node, installer_node, timout=120)
+    """
+    cmd = f"ifconfig {interface} down;sleep {timeout};ifconfig {interface} up"
+    background_cmd = Thread(target=(lambda: node.exec_command(cmd=cmd, sudo=True)))
```

Comment:
`thread` would be more meaningful variable name here

Reply:
changed to thread